src/Features/Core/Diagnostics/EngineV1/DiagnosticAnalyzerDriver.cs

```diff
@@ -479,11 +479,14 @@ public bool IsAnalyzerSuppressed(DiagnosticAnalyzer analyzer)
 
                 if (hasDependentCompilationEndAction && forceAnalyzeAllDocuments != null)
```

Comment:
Now that it is A) not possible to register a compilation end action in Initialize, and B) were it possible to register one, such an action would be treated as equivalent to one registered in a compilation start action, I think this mechanism can be simplified by returning it to a prior state. If this proves to be so, either @mavasani or I can do this in a later change.